tests/data/parity/trace4.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "action": {
+      "from": "0x2a65aca4d5fc5b5c859090a6c34d164135398226",
+      "callType": "call",
+      "gas": "0x7148",
+      "input": "0x",
+      "to": "0x379add715d9fb53a79e6879653b60f12cc75bcaf",
+      "value": "0xe372bcabcbd7c00"
+    },
+    "blockHash": "0x75095f0aac09ad8850bfeaee60802e28402da64f728941bb3d8c2d54bf82425f",
+    "blockNumber": 5762231,
+    "result": {
+      "gasUsed": "0x22f0",
+      "output": "0x0000000000000000000000000000000000000000000000000000000000000000"
+    },
+    "subtraces": 1,
+    "traceAddress": [],
+    "transactionHash": "0x23c21f0cc34ef5ab31196ba4afce76723cf3614e53807f6a6bf094c6d2125bfe",
+    "transactionPosition": 154,
+    "type": "call"
+  },
+  {
+    "action": {
+      "from": "0x379add715d9fb53a79e6879653b60f12cc75bcaf",
+      "callType": "callcode",
+      "gas": "0x530e",
+      "input": "0x",
+      "to": "0x273930d21e01ee25e4c219b63259d214872220a2",
+      "value": "0xe372bcabcbd7c00"
+    },
+    "blockHash": "0x75095f0aac09ad8850bfeaee60802e28402da64f728941bb3d8c2d54bf82425f",
+    "blockNumber": 5762231,
+    "result": {
+      "gasUsed": "0x5cc",
+      "output": "0x"
+    },
+    "subtraces": 0,
+    "traceAddress": [
+      0
+    ],
+    "transactionHash": "0x23c21f0cc34ef5ab31196ba4afce76723cf3614e53807f6a6bf094c6d2125bfe",
+    "transactionPosition": 154,
+    "type": "call"
+  }
+]
```

Comment:
Can we add newlines to the ends of these files? GH shows the red circle per file where its missing